.github/workflows/e2e.yaml

```diff
@@ -28,11 +28,10 @@ jobs:
        - "1.19"
        - "1.27"
        group:
-         - e2e e2e-upgrade
+         - e2e e2e-upgrade e2e-prometheuscr
```

Comment:
Can we separate `e2e-prometheuscr` into its own section in the matrix?  `e2e e2e-upgrade` already has a noisy neighbor problem and I'd prefer not to add another.

Reply:
Sure. Just didn't want to separate in case somebody though it was not ok to create a new section.

Reply:
Luckily we have the CI checks setup to look at the dependent job, so we can now scale this worker job as needed.